src/main/resources/application.conf

```diff
@@ -1,5 +1,6 @@
 interop-api-gateway {
   port = 8088
+  baseUrl =  ${BASE_URL}
```

Comment:
I think this is not used.

Reply:
It is indeed not used but @galales suggested to include it in order to make the service not start in case the `BASE_URL` variable isn't declared in the env in order to prevent this problem from happening again.

Reply:
Ok, nice hint!